### ISSUE TITLE
JInsight should not fail if config file is missing

### ISSUE THREAD

**Issue by talonx** [ApptuitAI/JInsight#52]

**Comment by rshivane** [comment]
If the jinsight.config option is provided, but the file is missing - it is an error
If the jinsight.config option is not provided, and the default file is missing - it is an not an error

**Comment by rshivane** [comment]
> If the jinsight.config option is not provided, and the default file is missing - it is an not an error
This is not true. It is failing if the jinsight.config option is not provided, and the default file is missing
